### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by ForthtuN** [A-tG/VoicemeeterFancyOSD#6] [General]
Not a bug or anything, but just so you know, really appreciate this "plugin". Been using it for some months now and its sad that original voicemeeter doesen't come with this or use this. This one is a must.

**Comment by terenc3** [comment]
I completely agree. Awesome work. I've setup hotkeys for my main lanes in VoiceMeeter (shift/alt/ctrl + vol keys) and having the OSD just fits perfectly
